Add image generation pipeline reuse into README (#2701)
openvinotoolkit/openvino.genai: samples/python/image_generation/README.md

Warmup number:1, first generate warmup time:85008.00 ms, infer warmup time:84999.88 ms
Generate iteration number:3, for one iteration, generate avg time: 84372.34 ms, infer avg time:84363.95 ms, all text encoders infer avg time:76.67 ms, vae encoder infer avg time:0.00 ms, vae decoder infer avg time:4470.33 ms
```

### Image Generation Pipeline reuse

> To extend pipeline's capability, we do have an interface to allow a specific image generation pipeline to reuse models from another image generation pipeline which have already loaded the models, and below table shows the support scope.

Reviewer: ```suggestion
To extend the pipeline’s capabilities, we provide an interface that allows a specific image generation pipeline to reuse models from another pipeline that has already loaded them. The table below shows the support scope.
```

| Image Generation pipeline | Model can be reused from |
|:---|:---|
| `Text2ImagePipeline` | `Image2ImagePipeline` or `InpaintingPipeline` |
| `Image2ImagePipeline` | `InpaintingPipeline` |
| `InpaintingPipeline` | `Image2ImagePipeline` |

> This example shows how `Text2ImagePipeline` reuses models from `Image2ImagePipeline` and execute different pipeline depending on whether an initial image is provided.

Reviewer: ```suggestion
This example shows how `Text2ImagePipeline` reuses models from `Image2ImagePipeline` and executes a different pipeline depending on whether an initial image is provided.
```
